packages/smart-contracts/src/lib/ethereumProxyArtifact.ts

```diff
@@ -3,10 +3,15 @@ const ARTIFACTS_VERSION: string = artifactsEthereumProxy.lastVersion;
 
 /**
  * Retrieve the abi from the artifact of the used version
+ * @param version version of the contract
  * @returns the abi of the artifact as a json object
  */
-export function getContractAbi(): any {
-  const artifactFilename: string = artifactsEthereumProxy[ARTIFACTS_VERSION].artifact;
+export function getContractAbi(version?: string): any {
```

Comment:
I'm affraid of this breaking on the browser because of webpack. Did you/Can you test it?